Formula/d/dosbox-staging.rb

```diff
@@ -51,7 +51,6 @@ class DosboxStaging < Formula
   fails_with gcc: "5"
 
   def install
-    (buildpath/"subprojects").rmtree # Ensure we don't use vendored dependencies
```

Comment:
@unjello we cannot do this, as we dont want to use bundled dependencies.

Reply:
I'm not a meson expert, but looking at the build script

```if host_machine.system() == 'darwin'
    # use SDL2 wrap so we can control MACOSX_DEPLOYMENT_TARGET during the build
    sdl2_dep = subproject(
        'sdl2',
        default_options: default_wrap_options,
    ).get_variable('sdl2_dep')
else
    sdl2_dep = dependency(
        'sdl2',
        version: ['>= 2.0.5', '< 3'],
        static: ('sdl2' in static_libs_list),
        fallback: [],
        include_type: 'system',
    )
endif```

it look to me like that they've hardcoded bundled dependency for macOS, and some major patching of build system would be required not to use it (it was changed on purpose a month ago https://github.com/dosbox-staging/dosbox-staging/commit/e1dd3e283572d54b051f438caecf450c281e0130). I don't think I am up for the task of mediating between two opposite philosophies. Feel free to close this PR.

Reply:
Could you make an issue upstream linking here? Closing the PR will be done automatically if we can't figure it out otherwise.

Reply:
Sure, I'll poke dosbox-staging team. Thanks.